templates/agenda-for-all-hands-call.md

```diff
@@ -20,4 +20,8 @@
 - Demos
 - Moderator/notetaker for next time:
  * Moderator: TBD
- * Notetaker: TBD
\ No newline at end of file
+ * Notetaker: TBD
+
+<!-- After each call, it is the responsibility of the notetaker to save the last
+version of the notes in a file in ipfs/pm/meeting-notes, by opening a branch and
+submitting a PR. -->
```

Comment:
Yep. Mentioned that in https://github.com/ipfs/pm/issues/234#issuecomment-263315433. Once it is done, we can remove this.